docs/vtex-io/Storefront-Guides/google-tag-manager/vtex-io-documentation-setting-up-google-tag-manager.md

```diff
@@ -32,7 +32,7 @@ Before proceeding any further, make sure you have the following requisites:
 
 Once the GA4 Configuration tag is created, set up all GA4 events as follows:
 
-1. Download the [container file](https://gist.githubusercontent.com/filipewl/6fab5e75ae938487fe780b1ce213970f/raw/e9bf3db528bca7a07851512e378e53ac7d8ba08d/gtm-ga4-container-template.json). This container adds all the necessary tags, triggers, and variables.
+1. Download the [container file](/docs/vtex-io/Storefront-Guides/google-tag-manager/container-template.json). This container adds all the necessary tags, triggers, and variables.
```

Comment:
Updated the link with the correct version now!
```suggestion
1. Download the <a href="https://gist.githubusercontent.com/filipewl/6fab5e75ae938487fe780b1ce213970f/raw/3648454449222241dd2eccc3dc8b24a75f82d924/gtm-ga4-container-template.json" download>container file</a>. This container adds all the necessary tags, triggers, and variables.
```
I don't know if this markdown will understand the `a` tag, but it would be nice if we could use this [`download` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#:~:text=global%20attributes.-,download,-Causes%20the%20browser) to force a download when users click on the link.